src/frontend/src/pages/home/home.tsx

```diff
@@ -1,7 +1,119 @@
 import React from 'react';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import { Link } from "react-router-dom";
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Box from '@material-ui/core/Box';
+import Paper from '@material-ui/core/Paper';
+
 
 export const Home: React.FC = () => {
     return (
-        <div>Home Page</div>
+        <Box>
+
+            {/*     navigation bar      */}
+
+            <AppBar position="static">
+                <Toolbar>
+                    <Grid
+                        justify="space-between"
+                        alignItems="center"
+                        container
+                    >
+                        <Grid item >
+                            <IconButton edge="start" color="inherit">
+                                <AccountCircle />
+                            </IconButton>
+
+                            <Typography variant="h6" display="inline">
+                                0
+                            </Typography>
+                        </Grid>
+                        <Grid item>
+                            <Link to="/login" style={{ textDecoration: 'none' }}>
+                                <Button variant="contained" color="primary" size="large" >
+                                    Log Out
+                                </Button>
+                            </Link>
+                        </Grid>
+                    </Grid>
+                </Toolbar>
+            </AppBar>
+
+
+            {/* new challenge button    */}
+
+
+            <Grid
+                container
+                direction="column"
+                justify="center"
+                alignItems="center"
+                spacing={8}
```

Comment:
```suggestion
                spacing={3}
```
Spacing 8 seems a bit excessive. I've used spacing 2 in the new-challenge page, but spacing 3 also looks fine.